### PR TITLE
Fixed positioning in few areas and added Sponsor and About page

### PR DESCRIPTION
**Pull request by Hiten1708**
index.html - file
- Added some links for new font styles

App.js
- Added Sponsor Page 
- Added About Page

Navbar.js
- Added links in Navbar that will jump to the respective sections on click

Navbar.css
- Fixed some position of elements

Note: I have only included only one image for each sections in design because later when we have more umbrella imagery, we can assign the same class to a div that will contain all the images which will make positioning simpler

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,14 +2,19 @@ import './App.css';
...
     <div>
       <NavbarComp />
       <Home />
+      <About/>
+      <Sponsors/>
     </div>
   );
 }
 
+
```
Comment (Sneezi909) [inline src/App.js:19]: Can you remove this?

File: src/components/Home/home.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ function Home(){
 
                     {/* Heading */}
 
-                    <h1 className="quote">
+                    <h1 className="quote" >
```
Comment (Sneezi909) [inline src/components/Home/home.js:18]: Remove this space

File: src/App.js (hunk 1 above)
Comment (Hiten1708) [reply src/App.js:19]: Sure

**Review by FengchiW — APPROVED** [review]
Everything looks good for future tickets remove src/.DS_Store but for right now lets just get things in

**Comment by Hiten1708** [comment]
Resolves #4, resolves #5

**Pull request merged**